### ISSUE TITLE
a little bug about loss

### ISSUE THREAD

**Issue by Liu-Feng-deeplearning** [LAION-AI/CLAP#115]
At CLAP loss: logits_per_text = logit_scale_a * text_features @ audio_features.T
I think logit_scale_a should be logit_scale_t??

https://github.com/LAION-AI/CLAP/blob/776b7dc95f2dc71775e9dc9804876e23265285b7/src/laion_clap/clap_module/loss.py#L194C1-L195C1

Thanks a lot and Great work!!

**Comment by lukewys** [comment]
Hi, the naming of the variable is a bit confusing but we used an identical temperature scaling for both text and audio logits. Please also see: https://github.com/LAION-AI/CLAP/issues/109
